boards/arm/ip_k66f/pinmux.c

```diff
@@ -55,6 +56,11 @@ static int ip_k66f_pinmux_init(struct device *dev)
 	pinmux_pin_set(portb, 17, PORT_PCR_MUX(kPORT_MuxAlt2));
 #endif
 
+	struct device *gpioa = device_get_binding(DT_ALIAS_GPIO_A_LABEL);
```

Comment:
I do not think pinmux.c is the right place for that but probably board.c

Reply:
Ok. I will introduce board.c and add there the RESET gpio setup pins.